run_snafu.py

```diff
@@ -58,9 +58,13 @@ def main():
     #instantiate elasticsearch instance and check connection
     es={}
     if "es" in os.environ:
-        es['server'] = os.environ["es"]
-        es['port'] = os.environ["es_port"]
-        index_args.prefix = os.environ["es_index"]
+        if os.environ["es"] != ""
+            es['server'] = os.environ["es"]
+        if os.environ["es_port"] != ""
+            es['port'] = os.environ["es_port"]
+    if len(es.keys()) == 2:
+        if os.environ["es_index"] != "":
+            index_args.prefix = os.environ["es_index"]
```

Comment:
Good idea. Can we also throw out a log message? Something that lets the user know what server/port/index was used? (Yes they should know, but is always good to confirm)

Reply:
L72 will already be giving a full set of details, but updated the PR so that we're also showing info about what's server/port/index.